src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+(click: "rubber hits the road")[$dissolve[My jaw slacked and my lip quivered. I held my hand up and watched it tremble for a moment. But then, like Jay-sus settling the bubbles in a shook up bottle of Rock N Rye, my muscles went calm.]]
+
+(click: "calm")[$dissolve[I fixed myself into a battle stance and roared back, “I handle fine. I handle better! I ride steady and smooth like a ‘65 Cutlass at midnight with the headlights off and I’m COMING]]
+
+(click: "COMING")[THROUGH]
+
+(click: "THROUGH")[YOUR]
+
+(click: "YOUR")[FUCKING]
+
+(click: "FUCKING")[B U L L S H I T]
+
+(click: "B U L L S H I T")[$dissolve[I heard a crunching at the edge of my perception.]]
+
+(click: "crunching")[$dissolve[I looked down to see the sink starting to shift out of the wall. I had to scramble to keep it in place.(if: $sinkPuke is true)[ A bit of my puke sloshed onto my hand from the sink. Ugh.] I hadn’t realized how hard I’d been gripping it.]]
+
+(click: "gripping it")[$dissolve[The mirror stranger got me riled. She sure was a bitch, but she’s dead on. Funny how she’s fucking right as can be.
+
+(display: "The Gutter - Bathroom - Menu")]]
+
+::The Gutter - Bathroom - Toilet flashback
+(set: $toiletflashbackSeen to true)
+I sat down for a second. Just needed to clear my head. But, like everything else tonight, what I wanted didn’t happen. My ass pancaked onto the flimsy, plastic seat while my brain slipped beyond me, into the past.
+
+(click: "into the past")[$dissolve[I’m eleven years old. Anna and I are dancing while my mom watches from a lawn chair. It’s a summer day when it feels like you’re racing around a band of the sunshine that’s never gonna flatten out and end.]]
+
+(click: "flatten out and end")[$dissolve[Anna tugs me over to this group of guys that she’s been talking to. Leo and his dad, some others. We take our boombox over and do this dance routine we’ve been practicing for a few days, nothing fancy, but we really worked at it and it’s kinda long. There are a few parts where we pick each other up.]]
+
+(click: "really worked at it")[$dissolve[While we’re dancing, I don’t feel anything. I don’t feel the eyes. I don’t hear any hoots, if anyone was even drunk enough to hoot at two eleven year olds. We just twirl and twirl in infinity. I remember thinking: “I hope this lasts forever.”]]
+
+(click: "forever")[$dissolve[And it ends, of course. We’re both breathing heavily and beaming. The guys clap and Leo’s dad waddles over doing a bad copy of our smiles. He says, “Y’all are gonna be heartbreakers one of these days.”]]
+
+(click: "one of these days")[$dissolve[He pinches my cheek, pats my ass. His hands leave long, dark smudges of machine oil. I try to pull away, but he catches me.]]
+
+(click: "catches me")[$dissolve[He says, “Hope you’re nothing like your mother, darling” and loses his balance. He falls ass backwards and his beer’s in the air arcing like a sputtering hose. Everyone busts out laughing, especially Anna.]]
```

Comment:
Just a style suggestion, but I think this flows a little better:

"He falls ass backwards and his beer’s in the air arcing like a sputtering hose."
||
V
"He falls ass-backwards, spilling his beer into the air like a sputtering hose."

Mostly I just think "arcing" looks weird, but that might just be me. Also, yeah, "ass-backwards" should be hyphenated because it's a compound adverb modifying "falls."